k8s-daemonset/cert.pem

```diff
@@ -0,0 +1,62 @@
+-----BEGIN CERTIFICATE-----
+MIIFZzCCA0+gAwIBAgICEAAwDQYJKoZIhvcNAQELBQAwMzELMAkGA1UEBhMCVVMx
+EzARBgNVBAgMCkNhbGlmb3JuaWExDzANBgNVBAoMBm9saXgwcjAeFw0xNzAzMjIy
+MTIxNTZaFw0yNzAzMjAyMTIxNTZaMFIxCzAJBgNVBAYTAlVTMRMwEQYDVQQIDApD
+YWxpZm9ybmlhMQ8wDQYDVQQKDAZvbGl4MHIxHTAbBgNVBAMMFG9saXgwciBwbGF5
+Z3JvdW5kIGNhMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEApEMTTiTo
+LGZzxckwysVz3LcVleK9o3HBgkcUrDK0FhNyp6/RNOd85G/QVNpMP72a6LSTJMqL
+S4t2Q6hq3MosZIhvDr5gstEwrhnAzNbMLYZDE1F5VypLPG9RX9iK+VA9pP4MTHJj
+2ZE5zRbW44WmuglfutTgH/ifXVSFma/pZ6xvD6oc5cVzViG7hcXhMF8rfzlF9S5T
+NbLSYVT7mXJ8Hq+In+w47vCyS0WeCOTywsKoecqREmODBun91hLJip4ZVKebRwQu
+JeKMga/WXpUTbCV9rYL0YXlF3PH+jzxR35bodNtKs5BzM73Vh7lebya6r79efeZz
+uVL8/g6Qv2anHIUaa4f4fddny+lc0pe8KetBW5hSswBqiQyox+a1e9mNOf4y8G0T
+DGmze+vy/LL6h1l5TYruqvG6gU3/DdkaeNazdlXJAIoWzEOsIE2KzaD1+Covs9jG
+Srpgj2SfPCQtNwI6/qYlpLmpP1w4sDtdFhaW6U+HuHzUZM57/oE0AWvIRuumH/rU
+Bj4kJ9wW7DBKurgvCFz4HUf2Vfx5FrcohU9XBEeDL9+TsZKmTGidINzO/ne6Vzkt
+hMMdI3kfDsOt3NcuvyIb8/d908qx9SbDMA0na9YO05rBGUsIHSyn3+78UQ+a3QQ2
+RrIUI3zYUyw1vApv4l/8LnOWafAEYFyOqnUCAwEAAaNmMGQwHQYDVR0OBBYEFOZJ
+FJlHTbWQckw5AQzot/6Ai1uBMB8GA1UdIwQYMBaAFFeoSULQk8GwYMaOObgQ8uZL
+rHpUMBIGA1UdEwEB/wQIMAYBAf8CAQAwDgYDVR0PAQH/BAQDAgGGMA0GCSqGSIb3
+DQEBCwUAA4ICAQAp9v2nxd2YvROz71tDeQlBqNLLlOlDywG3DdHegccRo7DwwiaU
+U0RH8cLk8Ken4aHeRLUZSVsKE4SbfwxMCFUHq80+odSxspSo9ZhrEKyog7mF/H+W
+UUXqebw4JqKgdEQvrMNTYw9OlDwJrkQoQ8SPnfG7XilG0JU1YnjGaEhvCzOzgRxC
+4quXKqhfNGZvh1UubpgweZV4vRyoWhvbYo/GXmc8uO3b49iLoPvk7VHKhRKt9zvx
+j4y0bwZdrt2sSyRhKpioAEJRnc3/ow1UT81m5+IOIDg3mDPuwIcR7kprsQFfMsDK
+ONil3XWNkBNvRxmSis+ch0v94nKbktgOicD7/HGOxiXVsBNqw1OkhlNj7bj3LSbQ
+2GK4crpZppeIzzYgsoE2CeqXUf96XU3AdR3gsiep6ayM8eqa6E4OAXsDiqvJYE/i
+R3WxwoI1a6BnYX2B/31X9B5qxhTNrRU7p/DukSdXxaXgFiaImFJKUskCilhiXbyc
+eKsUgFUJAIgYZekdCcPqClwBZCak7x+3b6Pu+zBqLG/kF9mo7SL9vwa0bLe78dc+
+yvT5I0z5x8wbVVPxJttMK/sBoWMoVbfn9sJZjL42RDfCGev+TOAQ/QjyStgkZfeB
+rwf50sNwWpzH6bzUMGNU+FymoohpUPKp8hYd5oFJ5eGvXamiCCqccs5mrw==
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
```

Comment:
huh yeah not sure about the two certs thing, this is what I get when running https://github.com/olix0r/eg-ca/blob/master/mksvc.sh and I verified it works so :woman_shrugging: 

I like the subdir idea